packages/fx-core/src/question/create.ts

```diff
@@ -2420,8 +2425,7 @@ export function capabilitySubTree(): IQTreeNode {
             inputs[QuestionNames.Capabilities] !==
               CapabilityOptions.copilotPluginOpenAIPlugin().id &&
             inputs[QuestionNames.MeArchitectureType] !== MeArchitectureOptions.apiSpec().id &&
-            inputs[QuestionNames.Capabilities] !== CapabilityOptions.officeAddinImport().id &&
-            inputs[QuestionNames.Capabilities] !== CapabilityOptions.outlookAddinImport().id
+            inputs[QuestionNames.Capabilities] !== CapabilityOptions.officeAddinImport().id
```

Comment:
when capability = outlook-addin-import or office-addin-import, programming-language will not show